pytest_fixtures/api_fixtures.py

```diff
@@ -303,6 +305,34 @@ def module_azurerm_cloudimg(default_architecture, default_os, module_azurerm_cr)
     return finish_image
 
 
+@pytest.fixture(scope='module')
+def module_azurerm_gallery_finishimg(default_architecture, default_os, module_azurerm_cr):
+    """ Creates Shared Gallery Finish Template image on AzureRM Compute Resource """
+    finish_image = entities.Image(
+        architecture=default_architecture,
+        compute_resource=module_azurerm_cr,
+        name=gen_string('alpha'),
+        operatingsystem=default_os,
+        username=settings.azurerm.username,
+        uuid=AZURERM_RHEL7_FT_GALLERY_IMG_URN,
+    ).create()
+    return finish_image
+
+
+@pytest.fixture(scope='module')
+def module_azurerm_custom_finishimg(default_architecture, default_os, module_azurerm_cr):
```

Comment:
`module_azurerm_gallery_finishimg >> module_azurerm_gallery_finish_img`

Reply:
Using the same naming as others already have to keep consistent ..

https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/api_fixtures.py#L264

https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/api_fixtures.py#L27